src/controllers/user.controller.ts

```diff
@@ -0,0 +1,72 @@
+import userService from '../services/user.service'
+
```

Comment:
erase all blank rows in the file

Reply:
All blank rows make the code more readable, I think that they are necessary because separate the part of imports and functions